bin/_mocha

```diff
@@ -187,13 +187,44 @@ Error.stackTraceLimit = Infinity; // TODO: config
 
 var reporterOptions = {};
 if (program.reporterOptions !== undefined) {
-    program.reporterOptions.split(",").forEach(function(opt) {
-        var L = opt.split("=");
-        if (L.length != 2) {
-            throw new Error("invalid reporter option '" + opt + "'");
+
+   program.reporterOptions.split(",").forEach(
+
+    // Two possible formats ...
+     /\w+{(\w+:\w+)+}/.test(program.reporterOptions) ?
```

Comment:
Oh, the new syntax for `--reporter-options` has been added. Sorry, I missed that.